qbft/spectest/tests/prepare/prepare_quorum_triggered_twice.go

```diff
@@ -0,0 +1,71 @@
+package prepare
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	qbftcomparable "github.com/bloxapp/ssv-spec/qbft/spectest/comparable"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// PrepareQuorumTriggeredTwice is a spec test that checks the case where all prepares are sent and quorum event is triggered more than once.
+func PrepareQuorumTriggeredTwice() tests.SpecTest {
```

Comment:
what is the effect tested here? did we find a bug? commit message sent twice?

Reply:
The issue was that the audit team thought that core tests were missing.
They didn't specify a bug and on this case we didn't find a bug.

However, look at the net comment for a bug that I found :-)

Reply:
so this is purely audit feedback and fixes?